apps/rstudio/app.yaml

```diff
@@ -5,20 +5,12 @@ description: |
 version: 0.0.10
 services:
   rstudio:
-    # image: "bioconductor/bioconductor_docker:RELEASE_3_15"
     image: "us.gcr.io/broad-dsp-gcr-public/anvil-rstudio-bioconductor:3.15.3"
-    port: 8001
-    # inject a sed command to remove the www-address config used by Leo
+    port: 8787
```

Comment:
why change to 8787?

Reply:
I think the default base image is exposing 8787. But I'm not sure why it works on GCE via 8001